apollo-runtime/src/main/java/com/apollographql/apollo/internal/interceptor/ApolloServerInterceptor.java

```diff
@@ -88,7 +88,8 @@ public void interceptAsync(@NotNull final InterceptorRequest request, @NotNull f
         callBack.onFetch(FetchSourceType.NETWORK);
 
         try {
-          if (useHttpGetMethodForQueries && request.operation instanceof Query) {
+          if ((request.useHttpGetMethodForQueries || useHttpGetMethodForQueries)
```

Comment:
hm I guess you need to rebase as it should be only `request.useHttpGetMethodForQueries && request.operation instanceof Query`?

Reply:
Thanks for reviewing the PR.

As 'useHttpGetMethodForQueries' is passing through from ApolloClient as a global setting of using Get Method.
and
request.useHttpGetMethodForQueries is passing from previous interceptor chains. eg. the APQs interceptor.

so either one is true should use GET Method. Is it acceptable?

Reply:
ok get it, code is outdated, going to fix it.

Reply:
updated as above suggestion.